tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -2529,16 +2529,6 @@ telemetry/test_telemetry.py::test_telemetry_queue_buffer_cnt:
       - "topo_type in ['m0', 'mx', 'm1', 'm2', 'm3']"
       - "(is_multi_asic==True) and https://github.com/sonic-net/sonic-mgmt/issues/15393"
 
-#######################################
-#####         pktgen              #####
-#######################################
-test_pktgen.py:
-  skip:
-    reason: "Have known issue, only running for 'cisco-8000', skipping for all other ASIC types"
-    conditions:
-      - "asic_type not in ['cisco-8000']"
-      - "https://github.com/sonic-net/sonic-mgmt/issues/13804"
```

Comment:
Please check on different asic types too. The issue is not the only condition for skipping.